outboundSender.go

```diff
@@ -622,15 +622,8 @@ func (obs *CaduceusOutboundSender) send(urls *ring.Ring, secret, acceptType stri
 		Interval: obs.deliveryInterval,
 		Counter:  obs.deliveryRetryCounter.With("url", obs.id, "event", event),
 		// Always retry on failures up to the max count.
-		ShouldRetry: func(error) bool { return true },
-		ShouldRetryStatus: func(code int) bool {
-			for _, c := range obs.retryCodes {
-				if code == c {
-					return true
-				}
-			}
-			return false
-		},
+		ShouldRetry:       xhttp.ShouldRetry,
+		ShouldRetryStatus: xhttp.ShouldRetryStatus,
```

Comment:
@mtrinh11  the function name should be **RetryCodes** as the same is defined in  https://github.com/xmidt-org/webpa-common/blob/a59029ee5adce57ab9671e5efc0c817f3675d8a4/xhttp/retry.go#L187